.github/workflows/main.yml

```
@@ -82,3 +82,11 @@ jobs:
       - name: Mark the job as unsuccessful
         if: contains(needs.*.result, 'failure') || contains(needs.*.result, 'cancelled')
         run: exit 1
+  fmt:
+    name: Rustfmt
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

`checkout@v2` uses a deprecated version of node.

```suggestion
      - uses: actions/checkout@v4
```